src/sidebar/components/test/group-list-test.js

```diff
@@ -332,4 +332,58 @@ describe('groupList', function () {
     assert.calledWith($window.open, 'https://test.hypothes.is/groups/new',
       '_blank');
   });
+
+  it('should set showGroupMenu=false if third party service and only one group', function () {
```

Comment:
Component controllers are essentially classes. The convention for tests for class methods is to group tests for each method in a describe block:

```
describe('#showGroupMenu', () => {
  it('should return false if ...'), () => {

  });
});
```